app/src/main/java/com/tdtu/mywallet/MainMenu.java

```diff
@@ -127,7 +127,7 @@ protected void onResume() {
     }
 
     private void updateRecyclerViewData() {
-        List<ExpenseDaily> updatedData = dbHelper.getExpensesMadeToday();
+        List<ExpenseDaily> updatedData = expenses;
```

Comment:
The bug of the expenses menu not showing up due to the menu being called via `onResume()` is still present. You should write it with:

```java
    List<ExpenseDaily> updatedData = dbHelper.getExpensesMadeToday();
```